ViewModel/Utilities/ImportFromFDA1Helper.cs

```diff
@@ -444,11 +444,12 @@ private static AnalyticalFrequencyElement CreateManualAnalyticalElement(Probabil
 
             //there will be no analytical flows. We just need 
             List<double> analyticalFlows = new List<double>();
-            List<double> graphicalFlows = new List<double>();
+            GraphicalVM graphicalVM = new GraphicalVM("Graphical Flow Frequency", "Probability", "Flow");
```

Comment:
Could use string constants from the StringConstants class

Reply:
Using string constants now.